reana_client/cli/workflow.py

```diff
@@ -886,6 +888,21 @@ def add_verbose_data_from_response(response, verbose_headers, headers, data):
     multiple=True,
     help="Filter job logs to include only those steps that match certain filtering criteria. Use --filter name=value pairs. Available filters are compute_backend, docker_img, status and step.",
 )
+@click.option(
+    "--follow",
```

Comment:
Not sure if I understand the reason for serving logs from the database instead of OpenSearch? If we want to display finished logs in old format (the one users currently see), we could just check workflow status in CLI and display it the old way if workflow has completed. And in this case it is irrelevant from where the logs are retrieved.

If it is about logs retrieval and not about displaying, in case of errors, OpenSearch logs would usually contain more information, so if we always serve the logs of completed workflows from the database, we lose the benefit of more reliable and informative OpenSearch logs. And the change has to be done in `r-workflow-controller`, `reana-client` is not involved here.

Reply:
During standup decided to add a message.